packages/resin-event-log/test/resin-event-log.spec.js

```diff
@@ -114,8 +116,33 @@ function createGaMock(endpoint) {
 	return aggregateMock(mocks)
 }
 
+function validateGsQuery(event) {
+	return (
+		event.site_token === GOSQUARED_ID &&
+		event.api_key === GOSQUARED_API_KEY
+	)
+}
+
+function createGsMock(options, times) {
+	times = times || 1
+
+	_.defaults(options, {
+		host: GOSQUARED_HOST,
+		method: 'POST',
+		filterQuery: validateGsQuery,
+		filterBody: function() { return true },
+		response: '1'
+	})
+	delete options.event
```

Comment:
Ah I was passing the expected event to check if it was in the querystring, because the browser passes it that way. But in node it passes it as the request body, I didn't clean up properly when removing the browser tests.